helm/cas-provision/templates/airflowOAuth.yaml

```diff
@@ -6,6 +6,17 @@ metadata:
   labels: {{ include "cas-provision.labels" . | nindent 4 }}
```

Comment:
This all looks good. I see at the top of the file (not changed in this PR) we have an `if and..` I don't think we need the `and` since it's just one condition we're checking.